src/client/datascience/notebook/creation/notebookCreator.ts

```diff
@@ -45,6 +48,7 @@ export class NotebookCreator {
             label: JVSC_EXTENSION_DisplayName
         });
         const placeHolder = DataScience.placeHolderToSelectOptionForNotebookCreation();
+        traceInfoIf(IS_CI_SERVER, `Display quick pick for creation of notebooks ${items.length}`);
```

Comment:
Damn it, will remove this in another PR.

Reply:
@DonJayamanne  it's fine I think that I need to remove them all as a code-health issue. I don't believe it matters now. I'll file an issue.